README.md

```
@@ -4,946 +4,239 @@
+[![SwiftPM](https://img.shields.io/badge/SPM-Linux%20%7C%20iOS%20%7C%20macOS%20%7C%20watchOS%20%7C%20tvOS%20%7C%20Windows%20%7C%20Android%20%7C%20Wasm-success?logo=swift)](https://swift.org)
 [![Twitter](https://img.shields.io/badge/twitter-@brightdigit-blue.svg?style=flat)](http://twitter.com/brightdigit)
 ![GitHub](https://img.shields.io/github/license/brightdigit/Spinetail)
 ![GitHub issues](https://img.shields.io/github/issues/brightdigit/Spinetail)
 
-![GitHub Workflow Status](https://img.shields.io/github/workflow/status/brightdigit/Spinetail/Spinetail?label=Actions&logo=github)
-[![Bitrise](https://img.shields.io/bitrise/b2595eab70c25d1b?logo=bitrise&?label=bitrise&token=rHUhEUFkU2RUL-KGmrKX1Q)](https://app.bitrise.io/app/b2595eab70c25d1b)
+[![Spinetail](https://github.com/brightdigit/Spinetail/actions/workflows/Spinetail.yml/badge.svg)](https://github.com/brightdigit/Spinetail/actions/workflows/Spinetail.yml)
+[![CodeQL](https://github.com/brightdigit/Spinetail/actions/workflows/codeql.yml/badge.svg)](https://github.com/brightdigit/Spinetail/actions/workflows/codeql.yml)
+[![codecov](https://codecov.io/gh/brightdigit/Spinetail/branch/main/graph/badge.svg)](https://codecov.io/gh/brightdigit/Spinetail)
 
 [![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2Fbrightdigit%2FSpinetail%2Fbadge%3Ftype%3Dswift-versions)](https://swiftpackageindex.com/brightdigit/Spinetail)
```

markdownlint-cli2 (0.23.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)